lib/native/source/libboolector/interface_wrap.c

```diff
@@ -3452,13 +3475,13 @@ SWIGEXPORT jstring JNICALL Java_org_sosy_1lab_java_1smt_solvers_boolector_BtorJN
       if (!arg2) return 0;
     }
 	
-    FILE *f = 0;
-    f = fopen("temp", "w+");
-	if(f==NULL) {
+    FILE *fparse = 0;
+    fparse = fopen("tempparse", "w+");
```

Comment:
@baierd Can we use a temporary file (maybe via "tmpfile()") here instead of creating a real file? The current file remains on disk and might look strange for users.

Reply:
@kfriedberger 
Pretty sure we can, i will take a look.